src/index.html

```diff
@@ -348,8 +346,67 @@ <h2 class="social-media-links__title">Let&apos;s connect</h2>
       </section>
 
       <!-- Contact Form -->
-      <section class="container">
-        <!-- Social Media Links -->
+      <section class="container contact-me">
+        <h2 class="contact-me__title">Contact me</h2>
+
+        <p class="contact-me__description">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it!
+        </p>
+
+        <form
+          class="contact-me__form"
+          action="https://formspree.io/f/mqkonbkk"
+          method="POST"
+        >
+          <div>
+            <label class="contact-me__form__label" for="full-name">
+              Full name:</label
+            >
+            <input
+              class="contact-me__form__input"
+              type="text"
+              id="full-name"
+              name="fullName"
+              placeholder="Alexandra Loremipsum"
+              maxlength="30"
+              required
+            />
+          </div>
+          <div>
+            <label class="contact-me__form__label" for="email">Email: </label>
+            <input
+              class="contact-me__form__input"
+              type="email"
+              id="email"
+              name="email"
+              placeholder="alexandra@gmail.com"
+              required
+            />
+          </div>
+          <div>
+            <label class="contact-me__form__label" for="message">
+              Message:
+            </label>
+            <textarea
+              class="contact-me__form__input contact-me__form__textarea"
+              name="message"
+              id="message"
+              placeholder="Write your message here"
+              maxlength="500"
+            ></textarea>
```

Comment:
- Kindly consider using `aria-label` attribute instead of wrapping the input element with a `label` tag that has text but won't display because it's not on the `Figma` design😄. The `aria-label` attribute provides a text alternative for screen reader users to understand the purpose of the input element. Therefore, in this case, it would be more appropriate to use `aria-label` attribute on the input element instead of a label with text. but won't display😄. You can learn more about using `aria-label` [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label)